pkg/common/peertracker/getpid.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _GETPID_H
+#define _GETPID_H

```

Comment:
super duper nitpick: technically single underscore followed by an uppercase letter is reserved
```suggestion
#ifndef GETPID_H_
#define GETPID_H_

```